### ISSUE TITLE
Hubitat does not log or notify of Hubduino Child contact sensor device status change

### ISSUE THREAD

**Issue by scottmil** [DanielOgorchock/ST_Anything#195]
Daniel-Thank you so much for the ST_Anything/Hubduino amazing code...I have successfully migrated an ST Thingshield device from ST to HE with your help.  
Although my three Hubduino Child devices (contact1, switch1, switch2) respond correctly to commands and events in HE Dashboards, they are not being recognized in Hubitat Logs or Apps.  The attached files show my three child devices, and the debug log of the Child contact1 (Main Door Lock) sensor.  
As stated previously, change in status of contact1 does not appear in the HE Logs, nor are contact1 status changes recognized by my Notifier app.  When I configured the Notifier app, the app configuration did allow me to to include contact1 (Main Door Lock) as a trigger.
<img width="1199" alt="Hubduino contact1 logging" src="https://user-images.githubusercontent.com/10510570/72125198-8f082d00-331c-11ea-9ebd-c17d6eb84eac.png">
<img width="1253" alt="Hubitat Hubduino nDevice" src="https://user-images.githubusercontent.com/10510570/72125209-97f8fe80-331c-11ea-9b8e-185b6d39c31f.png">

**Comment by ogiewon** [comment]
I am not really seeing the problem, yet.  You’re showing live log data, which clearly shows the hub is receiving data from the Arduino.  With debug logging enabled on the child device, it shows the regular 5 minute updates that occur whether or not the sensor status has changed. This looks fine to me.  The debug logging turns itself back off after 30 minutes, which is also shown in the logs.  If the status has not changed, then no event will be generated by the hub, and thus the Notifications App won’t send anything.  If you go into the device details page for the child device, and click EVENTS at the top, that may help to explain what is going on.

Are you opening and closing the door during the above timeframes that are shown above?

I haven’t hooked up a ThingShield for quite some time, so there is a chance something may have changed. But looking at the above Live Logs, I am not seeing an obvious issue.

**Comment by scottmil** [comment]
My apologies...my Hubduino ThingShield child devices are working properly.

I have two contact sensors on my garage door: an Aeotec Door/Window sensor (Main Door Sensor) to notify when the door is in the open position and a ThingShield child sensor (Main Door Lock) to notify if the door is unlocked: both sensors are triggers in my HE Notifications app if either is open for 10 minutes or more.   

I mistakenly thought the ThingShield sensor was failing to notify after 10 minutes, not realizing that the Aeotec sensor was also triggering at the same time, with the HE Pushover notification only including the description Main Door Sensor in the message.   When I tested the ThingShield to trigger the Notifications app by itself, the Main Door Lock description was included in the push message.  (attached Logs screenshot)

Being new to HE, I also failed to realize that not all devices (apparently including the ThingShield child devices) are capable of having a descriptionText in the HE Logs.   Even though my Hubduino ThingShield child devices were showing correct functioning in the HE Dashboards, I was initially puzzled why I was not seeing any ThingsShield messages in the Logs.

Again, I appreciate all of your work to make the transition from ST to HE so effortless.

<img width="1177" alt="Child contact1-Main Door Lock" src="https://user-images.githubusercontent.com/10510570/72185180-0f737000-33a7-11ea-9dfd-a6d6e7eeae8e.png">
